Backend/src/test/java/idealab/api/operations/PrintJobOperationsTest.java

```diff
@@ -398,6 +397,45 @@ public void createNewPrintJobSaveAllNonExistentData() {
         assert(opResponse.equals(response));
     }
 
+    @Test(expected = IdeaLabApiException.class)
+    public void createNewPrintJobWithNotFoundColor() {
```

Comment:
As you know, all unit tests have three phases. They are: arrange (given), act (when), assert (then). Could you comment on those phases' starting points to that it could be easily stood out?

        // given
        PrintJobNewRequest request = new PrintJobNewRequest();
        request.setColor("RED");
        request.setComments("COMMENTS");
        request.setCustomerFirstName("test");
        request.setCustomerLastName("testLast");
        request.setEmail("test@email.com");
        request.setFile(file);

        CustomerInfo customerInfo = new CustomerInfo();
        customerInfo.setFirstName("test");

        ColorType color = new ColorType();
        color.setColor("RED");
        color.setAvailable(true);

        Employee e = new Employee();
        e.setId(999);

        EmailHash emailHash = new EmailHash();
        emailHash.setEmailHash("test@email.com");

        Map<String, String> data = new HashMap<>();
        data.put("filePath", "DROPBOX_PATH");
        data.put("sharableLink", "http://testlink.com");

        when(emailHashRepo.findByEmailHash(any())).thenReturn(null);
        when(emailHashRepo.save(any())).thenReturn(emailHash);
        when(customerInfoRepo.findByEmailHashId(any())).thenReturn(null);
        when(customerInfoRepo.save(any())).thenReturn(customerInfo);
        when(colorTypeRepo.findByColor(any())).thenReturn(null);
        ...